### PR TITLE
Add the user.toggle_presence_by_number action.

### PR DESCRIPTION
**Pull request by reiterl**
Add some tests.
In the issue the action was renamed to toggle_presence_by_number.
The action is near the set_present action.

### CODE REVIEW

File: openslides_backend/action/actions/user/toggle_presence_by_number.py [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+            raise ActionException("Found more than one user with the number.")
+
+        filter_ = And(
+            FilterOperator(f"number_${meeting_id}", "=", ""),
+            FilterOperator("default_number", "=", number),
+        )
+        result = self.datastore.filter(Collection("user"), filter_, ["id"])
+        if len(result.keys()) == 1:
+            return list(result.keys())[0]
+        raise ActionException("Could not find only one user with the number.")
```
Comment (jsangmeister) [inline openslides_backend/action/actions/user/toggle_presence_by_number.py:78]: Differentiate here between the cases "No user found" and "Too many users found" for better error messages

File: tests/system/action/user/test_toggle_presence_by_number.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
+from openslides_backend.permissions.management_levels import (
+    CommitteeManagementLevel,
+    OrganizationManagementLevel,
+)
+from openslides_backend.permissions.permissions import Permissions
+from tests.system.action.base import BaseActionTestCase
+
+
+class UserSetPresentActionTest(BaseActionTestCase):
```
Comment (jsangmeister) [inline tests/system/action/user/test_toggle_presence_by_number.py:9]: Please also test the failure cases (too many/no users with a certain number, too many/no users with a certain default number) and edge cases (if a user with a certain number exist and another user has it as the default number, only the first one is toggled; if a user has a normal number and a default number, the default number is ignored, even if it matches)

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/actions/user/toggle_presence_by_number.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/user/toggle_presence_by_number.py]: Diff the two cases.

File: tests/system/action/user/test_toggle_presence_by_number.py (hunk 2 above)
Comment (reiterl) [reply tests/system/action/user/test_toggle_presence_by_number.py]: Add some more tests. 3 failure tests and 2 edge tests.

**Review by jsangmeister — APPROVED** [review]
